app/src/main/java/com/bnyro/contacts/ui/components/shapes/CurlyCornerShape.kt

```diff
@@ -0,0 +1,84 @@
+package com.bnyro.contacts.ui.components.shapes
+
+import androidx.compose.foundation.shape.CornerBasedShape
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.shape.ZeroCornerSize
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Outline
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.unit.LayoutDirection
+import kotlin.math.cos
+import kotlin.math.sin
+
+val curlyCornerShape = CurlyCornerShape()
+
+class CurlyCornerShape(
+    private val amp: Double = 10.0,
+    private val count: Int = 8,
+) : CornerBasedShape(
+    topStart = ZeroCornerSize,
+    topEnd = ZeroCornerSize,
+    bottomEnd = ZeroCornerSize,
+    bottomStart = ZeroCornerSize
+) {
+
```

Comment:
```suggestion
object CurlyCornerShape : CornerBasedShape(
    topStart = ZeroCornerSize,
    topEnd = ZeroCornerSize,
    bottomEnd = ZeroCornerSize,
    bottomStart = ZeroCornerSize
) {

    private const val amp: Double = 10.0
    private const val count: Int = 8
```

Reply:
We might want to use this at different places too in the future, so I think we should keep it (We need to modify these values when we change the size of the shape in order for it to stay good-looking)